### PR TITLE
[permissions] Add permissions for qr service. Contributes to JB#49377

### PR DESCRIPTION
**Pull request by HandsomeJaack**

### CODE REVIEW

File: permissions/Qr.permission [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-qr
+# x-sailjail-description = QR
+# x-sailjail-translation-key-long-description = permission-la-qr_description
+# x-sailjail-long-description = Recognize QR codes
```
Comment (rainemak) [inline permissions/Qr.permission:7]: Recognize QR codes... QR code reading

Comment (okodron) [reply permissions/Qr.permission:7]: Indeed

Comment (rainemak) [reply permissions/Qr.permission]: On Android camera permissions it is "Allow QR code reader" but that doesn't of the version and provider. We already dropped those "Allow" prefixes so please do not add that.

So my vote goes to "QR code reader" (not reading)

Comment (rainemak) [reply permissions/Qr.permission]: See above ^

File: permissions/Qr.permission [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# -*- mode: sh -*-
+
+# x-sailjail-translation-catalog = sailjail-permissions
+# x-sailjail-translation-key-description = permission-la-qr
+# x-sailjail-description = QR
```
Comment (rainemak) [inline permissions/Qr.permission:5]: Pondering about this short description... Somehow just "QR" sounds odd.

Comment (Tomin1) [reply permissions/Qr.permission:5]: I would maybe use these:
descripiton: QR codes
long-description: Read QR codes

**Review by okodron — APPROVED** [review]

File: permissions/Camera.permission [hunk 3]
```diff
@@ -11,3 +11,7 @@
 dbus-system.talk org.maemo.resource.manager
 dbus-system.broadcast org.maemo.resource.manager=org.maemo.resource.manager.*@/*
 # END systembus-org.maemo.resource.manager
+
+# BEG sessionbus-org.sailfishos.zxing
+dbus-user.talk org.sailfishos.zxing
```
Comment (rainemak) [inline permissions/Camera.permission:16]: No need to listen signal from "org.sailfishos.zxing"?

Comment (HandsomeJaack) [reply permissions/Camera.permission:16]: Works fine without it.

Comment (rainemak) [reply permissions/Camera.permission:16]: Ok, and if there are issues later it's easy to add "dbus-user.broadcast" rules for this as well.

**Review by rainemak — APPROVED** [review]

**Pull request merged**